### ISSUE TITLE
Esophageal hyperkeratosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10041]
**Preferred term label:**
Esophageal hyperkeratosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C5704568

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002031

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Plummer-Vinson syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think that this is not a separate phenotype, but rather it is a description of what you can observed with
[Esophageal web HP:0100594](https://hpo.jax.org/app/browse/term/HP:0100594).
I will add a description of esophageal hyperkeratosis to the comments of this term and add a reference to https://www.ncbi.nlm.nih.gov/books/NBK538306/
